merkle-tree/config/default.js

```diff
@@ -124,13 +51,9 @@ module.exports = {
   # Specify one of:
   # - 'remote' (to GET them from a remote microservice); or
   # - 'mongodb' (to get them from mongodb); or
-  # - 'compile' (to compile the contracts from /app/build to /app/build/contracts)
-  # - 'default' (to get them from the /app/build/contracts folder)
+  # - 'default' (to get them from the app/build/contracts folder)
   */
-  contractOrigin: process.env.CONTRACT_ORIGIN,
-
-  contractsPath: '/app/contracts/', // where to find contract .sol files (if applicable)
-  buildPath: '/app/build/contracts/', // where to find the contract interface json files
+  contractLocation: process.env.CONTRACT_LOCATION,
```

Comment:
`contractLocation` has been decremented (in a previous PR labelled as a breaking change). `contractOrigin` is the new term for this.

Reply:
https://github.com/EYBlockchain/timber/pull/3